clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/ComputeAggregationRequestBuilder.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.venice.client.store;
+
+import com.linkedin.venice.client.store.predicate.Predicate;
+import java.util.Map;
+
+
+public interface ComputeAggregationRequestBuilder<K> extends ExecutableRequestBuilder<K, ComputeAggregationResponse> {
```

Comment:
This `ComputeAggregationRequestBuilder` might not be very meaningful here, as it is just interface, we could add its implementation and its test after.